pywwt/core.py

```diff
@@ -66,6 +66,32 @@ def center_on_coordinates(self, coord, fov, instant=True):
                        fov=fov.to(u.deg).value,
                        instant=instant)
 
+    local_horizon_mode = Bool(False, help='Whether the view should be that of a local latitude, longitude, and altitude').tag(wwt='localHorizonMode', sync=True)
+    location_altitude  = AstropyQuantity(0 * u.m, help='Assigns altitude (in meters) for view location').tag(wwt='locationAltitude', sync=True)
+    location_latitude  = AstropyQuantity(47.633 * u.deg, help='Assigns latitude for view location').tag(wwt='locationLat', sync=True)
+    location_longitude = AstropyQuantity(122.133333 * u.deg, help='Assigns longitude for view location').tag(wwt='locationLng', sync=True)
+    
+    @validate('location_altitude')
+    def _validate_altitude(self,proposal):
+        if proposal['value'].unit.physical_type == 'length':
+            return proposal['value'].to(u.meter)
+        else:
+            raise TraitError('location_altitude not in units of length')
+   
+    @validate('location_latitude')
+    def _validate_latiitude(self,proposal):
+        if proposal['value'].unit == u.degree:
+            return proposal['value']
```

Comment:
You can do the same as for the altitude here: check the physical type is 'angle' then if so convert to degrees - it's unlikely anyone will give the lon/lat in other units, but you never know...